static/test/javascripts-legacy/spec/common/football.spec.js

```diff
@@ -0,0 +1,118 @@
+define([
+    'helpers/injector'
+], function (Injector) {
+
+    describe('Football', function () {
+        var injector = new Injector();
+        var sut;
+        var sandbox;
+        var destroySpy;
+
+        function mockMatchListLive() {}
+        
+        beforeEach(function (done) {
+            document.body.innerHTML = '<img class="media-primary"></img><div class="js-football-meta"></div>';
+
+            sandbox = sinon.sandbox.create();
+
+            destroySpy = sandbox.spy();
+
+            mockMatchListLive.prototype.destroy = destroySpy;
```

Comment:
These three lines:

```
sandbox = sinon.sandbox.create();
destroySpy = sandbox.spy();
mockMatchListLive.prototype.destroy = destroySpy;
```
can probbaly be moved outside the `beforeEach`. There's no need to repeat them, because the `sandbox` is being `restore`d in the `afterEach`